api/api/models/image.py

```diff
@@ -108,6 +108,13 @@ class Meta:
 
 
 class ImageReport(AbstractMediaReport):
+    """
+    Represents images that have been reported by users.
```

Comment:
Editorial nit, though stronger on this one. The issue of the current phrasing is a suggestion that the model represents something about an image (or work generally, extrapolated to audio report), whereas it represents the user-submitted report of an image. The same issue exists on audio report.

```suggestion
    User-submitted report of an image.
```